packages/api/package.json

```diff
@@ -5,7 +5,7 @@
   "main": "src/index.js",
   "scripts": {
     "lint": "eslint .",
-    "pretest": "npm run lint && npm run prettier",
```

Comment:
I've started removing these wherever I see them because our ESLint config already runs Prettier, so this is always going to run Prettier on an already prettified codebase.